extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -216,22 +211,19 @@ function Autoplay({
           onClick={onOverlayClick}
         ></div>
       )}
-
-      {/* TODO(wg-bento): Global styling.
-          https://github.com/ampproject/wg-bento/issues/7 */}
-      <style>{autoplayCss}</style>
     </>
   );
 }
 
 /**
  * @return {!PreactDef.Renderable}
  */
-const AutoplayIconContent = once(() =>
-  [1, 2, 3, 4].map((i) => (
-    <div className="amp-video-eq-col" key={i}>
-      <div className={`amp-video-eq-filler amp-video-eq-${i}-1`}></div>
-      <div className={`amp-video-eq-filler amp-video-eq-${i}-2`}></div>
+const AutoplayIconContent = once(() => {
+  const classes = useStyles();
+  return [1, 2, 3, 4].map((i) => (
+    <div className={`${classes.eqCol} amp-video-eq-col`} key={i}>
+      <div className={`amp-video-eq-filler ${classes[`eq-${i}-1`]}`}></div>
+      <div className={`amp-video-eq-filler ${classes[`eq-${i}-2`]}`}></div>
```

Comment:
@samouri Do I need to unwrap this loop since the member access for `classes` is dynamic?

Reply:
(Current compiled output seems ok btw. In the future we could optimize this away as to not read `classes`, which would then break this.)

Reply:
Context: I have a PR coming up that concerts the `classes` object to an enum.

You don't _need_ to unroll the loop, but if you don't it'll deopt. I just tested it via the appspot closure compiler.

**[static access (unrolled)](https://closure-compiler.appspot.com/home#code%3D%252F%252F%2520%253D%253DClosureCompiler%253D%253D%250A%252F%252F%2520%2540compilation_level%2520ADVANCED_OPTIMIZATIONS%250A%252F%252F%2520%2540output_file_name%2520default.js%250A%252F%252F%2520%2540formatting%2520pretty_print%250A%252F%252F%2520%2540language_out%2520ECMASCRIPT5_STRICT%250A%252F%252F%2520%253D%253D%252FClosureCompiler%253D%253D%250A%250A%250A%252F**%2520%2540enum%2520%257Bstring%257D%2520*%252F%250Aconst%2520a%2520%253D%2520%257B%2520cx1%253A%2520'hello'%252C%2520cx2%253A%2520'world'%257D%253B%250A%250Aconsole.log(a.cx1)%253B%250Aconsole.log(a.cx2))**
```js

// Source file
/** @enum {string} */
const a = { cx1: 'hello', cx2: 'world'};

console.log(a.cx1);
console.log(a.cx2)

// Compiled output
console.log("hello");
console.log("world");
```

**[dynamic access](https://closure-compiler.appspot.com/home#code%3D%252F%252F%2520%253D%253DClosureCompiler%253D%253D%250A%252F%252F%2520%2540compilation_level%2520ADVANCED_OPTIMIZATIONS%250A%252F%252F%2520%2540output_file_name%2520default.js%250A%252F%252F%2520%2540formatting%2520pretty_print%250A%252F%252F%2520%2540language_out%2520ECMASCRIPT5_STRICT%250A%252F%252F%2520%253D%253D%252FClosureCompiler%253D%253D%250A%250A%250A%252F**%2520%2540enum%2520%257Bstring%257D%2520*%252F%250Aconst%2520a%2520%253D%2520%257B%2520cx1%253A%2520'hello'%252C%2520cx2%253A%2520'world'%257D%253B%250A%250A%255B1%252C2%255D.forEach(i%2520%253D%253E%2520%257B%250A%2520%2520console.log(a%255B%2560cx%2524%257Bi%257D%2560%255D)%253B%250A%257D))**

```js
// Source file
/** @enum {string} */
const a = { cx1: 'hello', cx2: 'world'};

[1,2].forEach(i => {
  console.log(a[`cx${i}`]);
})

// Compiled output
var a = {a:"hello", b:"world"};
[1, 2].forEach(function(b) {
  console.log(a["cx" + b]);
});
```

Reply:
Yeah, we wanted to make it fully enumerable so that it can be fully DCE'd to constants.

Reply:
Instead of unrolling, I wrote a compound selector. I believe it can be DCE'd now.

Reply:
Or... the current solution to map only those classes one level above :)

Reply:
Or... just `nth-child(x):before, :after` :)